### PR TITLE
Change rollout settings for Fluentd to speed up rollout.

### PR DESCRIPTION
**Pull request by rene-dekker**
Fluentd takes ~1m to boot. With the old settings, a rollout of fluentd on a large cluster can take very long, while there is no need for fluentd to be available.

```
tigera-fluentd                 fluentd-node-nlf7f                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-r22cf                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-lx65p                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-x6b4f                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-nlf7f                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-r22cf                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-lx65p                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-x6b4f                              1/1     Terminating       8 (24m ago)     15d
tigera-fluentd                 fluentd-node-9j85c                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-rgcpr                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-x5hwr                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-9j85c                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-rgcpr                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-tpms6                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-x5hwr                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-tpms6                              0/1     Pending           0               0s
tigera-fluentd                 fluentd-node-9j85c                              0/1     ContainerCreating   0               0s
tigera-fluentd                 fluentd-node-rgcpr                              0/1     ContainerCreating   0               0s
tigera-fluentd                 fluentd-node-x5hwr                              0/1     ContainerCreating   0               1s
tigera-fluentd                 fluentd-node-tpms6                              0/1     ContainerCreating   0               1s
tigera-fluentd                 fluentd-node-9j85c                              0/1     ContainerCreating   0               1s
tigera-fluentd                 fluentd-node-rgcpr                              0/1     ContainerCreating   0               1s
tigera-fluentd                 fluentd-node-x5hwr                              0/1     ContainerCreating   0               1s
...
tigera-fluentd                 fluentd-node-s7cr8                              0/1     Running     0               45s
tigera-fluentd                 fluentd-node-tzwwh                              0/1     Running     0               45s
tigera-fluentd                 fluentd-node-v5qcs                              0/1     Running     0               45s
tigera-fluentd                 fluentd-node-wqjj9                              0/1     Running     0               45s
...
```

### CODE REVIEW

File: pkg/render/fluentd.go
Comment (caseydavenport) [inline pkg/render/fluentd.go:461]: Only concern is that this could break all of the nodes at once if a bad configuration is rolled out for some reason, whereas previously I think we would have stopped after rolling out one node and the other nodes would continue to function..

That said, I think it's probably better to just be faster in this case so I'm good with this.

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
